packages/block-library/src/quote/style.scss

```diff
@@ -1,4 +1,5 @@
 .wp-block-quote {
+	box-sizing: border-box;
```

Comment:
Applying box-sizing: border-box to the quote block might have a small impact on the themes frontend. But I expect that most themes already do this somehow, if not, this might fix some bugs if they add paddings to quotes... But it's still an impactful change we need to check.

(Similar to #43636 for lists)